docs/sources/fundamentals/getting-started/first-dashboards/get-started-grafana-prometheus.md

```diff
@@ -1,8 +1,9 @@
 ---
 aliases:
-  - ../guides/getting_started/
-  - ../guides/gettingstarted/
-  - getting-started-prometheus/
+  - ../../../guides/getting_started/ # /docs/grafana/latest/guides/getting_started
+  - ../../../guides/gettingstarted/ # /docs/grafana/latest/guides/gettingstarted
```

Comment:
More duplicates of `docs/sources/fundamentals/getting-started/first-dashboards/_index.md`
```suggestion
```